### PR TITLE
WebView2 landing update

### PR DESCRIPTION
**Pull request by JoshuaWeber**
Updates to the primary landing page for WebView2 for Build.

### CODE REVIEW

**Comment by JoshuaWeber** [comment]
The reference links broken in the releasenotes I think will be fixed in other PRs. I can update it on PRs to add in Versioning and Distribution docs.

File: microsoft-edge/webview2/index.md [hunk 1]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-## Microsoft Edge WebView2 SDK
+Hybrid applications allow developers to enjoy the best of both worlds. Hybrid application developers benefit from the ubiquity and strength of the web platform as well as the power and full capabilities of the native platform.
 
-To utilize WebView2 within your app you'll need to install and reference the [WebView2 SDK](https://aka.ms/webviewnuget) in your application. Our NuGet releases will contain both a release and pre-release version. The release version contains the public APIs we currently intend to release to general availability.
+## WebView2 Benefits 
+![webviewreasons](./media/WebView2/webviewreasons.PNG)
 
-The pre-release version will contain additional [experimental APIs](./reference/win32/0-9-488-reference-webview2.md#experimental). These are APIs and functionality we are evaluating and would like [feedback](https://aka.ms/webviewfeedback) on before promoting them to the release version.
+1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:37]:
```suggestion
1. Web ecosystem & skillset: utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem.
```

File: microsoft-edge/webview2/index.md [hunk 2]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+2. **Rapid Innovation:** web development allows for faster deployment and iteration
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:38]:
```suggestion
2. Rapid Innovation: web development allows for faster deployment and iteration.
```

File: microsoft-edge/webview2/index.md [hunk 3]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:39]:
```suggestion
3. Windows 7, 8, 10 support: support for a consistent user experience across Windows 7, 8, and 10.
```

File: microsoft-edge/webview2/index.md [hunk 4]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-To utilize WebView2 within your app you'll need to install and reference the [WebView2 SDK](https://aka.ms/webviewnuget) in your application. Our NuGet releases will contain both a release and pre-release version. The release version contains the public APIs we currently intend to release to general availability.
+## WebView2 Benefits 
+![webviewreasons](./media/WebView2/webviewreasons.PNG)
 
-The pre-release version will contain additional [experimental APIs](./reference/win32/0-9-488-reference-webview2.md#experimental). These are APIs and functionality we are evaluating and would like [feedback](https://aka.ms/webviewfeedback) on before promoting them to the release version.
+1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem
+2. **Rapid Innovation:** web development allows for faster deployment and iteration
+3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10 
+4. **Native Capabilities:** access the full set of Native APIs.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:40]:
```suggestion
4. Native Capabilities: access the full set of Native APIs.
```

File: microsoft-edge/webview2/index.md [hunk 5]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+5. **Code Sharing:** adding web code to your codebase allows for increased portability across multiple platforms
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:41]:
```suggestion
5. Code sharing: adding web code to your codebase allows for increased re-use across multiple platforms.
```

File: microsoft-edge/webview2/index.md [hunk 6]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+## WebView2 Benefits 
+![webviewreasons](./media/WebView2/webviewreasons.PNG)
 
-The pre-release version will contain additional [experimental APIs](./reference/win32/0-9-488-reference-webview2.md#experimental). These are APIs and functionality we are evaluating and would like [feedback](https://aka.ms/webviewfeedback) on before promoting them to the release version.
+1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem
+2. **Rapid Innovation:** web development allows for faster deployment and iteration
+3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10 
+4. **Native Capabilities:** access the full set of Native APIs.
+5. **Code Sharing:** adding web code to your codebase allows for increased portability across multiple platforms 
+6. **Microsoft Support:** When WebView2 GA’s it will have the full backing of Microsoft in terms of support and feature requests
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:42]:
```suggestion
6. Microsoft support: When WebView2 GA’s, Microsoft will provide support and add new feature requests.
```

File: microsoft-edge/webview2/index.md [hunk 7]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+7. **Evergreen Distribution:** rely on an up to date version of Chromium with regular and consistent platform updates and security patches
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:43]:
```suggestion
7. Evergreen distribution: rely on an up-to-date version of Chromium with regular platform updates and security patches.
```

File: microsoft-edge/webview2/index.md [hunk 8]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-The pre-release version will contain additional [experimental APIs](./reference/win32/0-9-488-reference-webview2.md#experimental). These are APIs and functionality we are evaluating and would like [feedback](https://aka.ms/webviewfeedback) on before promoting them to the release version.
+1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem
+2. **Rapid Innovation:** web development allows for faster deployment and iteration
+3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10 
+4. **Native Capabilities:** access the full set of Native APIs.
+5. **Code Sharing:** adding web code to your codebase allows for increased portability across multiple platforms 
+6. **Microsoft Support:** When WebView2 GA’s it will have the full backing of Microsoft in terms of support and feature requests
+7. **Evergreen Distribution:** rely on an up to date version of Chromium with regular and consistent platform updates and security patches 
+8. **Fixed Distribution (Coming Soon):** choice to package the Chromium bits in your application
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:44]:
```suggestion
8. Fixed distribution (coming soon): choice to package the Chromium bits in your application.
```

File: microsoft-edge/webview2/index.md [hunk 9]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+1. **Web Ecosystem & Skillset:** utilize the entire web platform, libraries, tooling, and talent that exists within the web ecosystem
+2. **Rapid Innovation:** web development allows for faster deployment and iteration
+3. **Windows 7, 8, 10 Support:** support for a consistent user experience across Windows 7, 8, and 10 
+4. **Native Capabilities:** access the full set of Native APIs.
+5. **Code Sharing:** adding web code to your codebase allows for increased portability across multiple platforms 
+6. **Microsoft Support:** When WebView2 GA’s it will have the full backing of Microsoft in terms of support and feature requests
+7. **Evergreen Distribution:** rely on an up to date version of Chromium with regular and consistent platform updates and security patches 
+8. **Fixed Distribution (Coming Soon):** choice to package the Chromium bits in your application 
+9. **Incremental Adoption:** add web components piece by piece into your application
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:45]:
```suggestion
9. Incremental adoption: add web components piece by piece to your application.
```

File: microsoft-edge/webview2/index.md [hunk 10]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+Web applications allow for a broad reach. Web developers can reuse most, if not all of their code, across all different platforms. Native applications, however, can utilize the capabilities of the entire native platform.
 
-## Microsoft Edge WebView2 SDK
+Hybrid applications allow developers to enjoy the best of both worlds. Hybrid application developers benefit from the ubiquity and strength of the web platform as well as the power and full capabilities of the native platform.
 
-To utilize WebView2 within your app you'll need to install and reference the [WebView2 SDK](https://aka.ms/webviewnuget) in your application. Our NuGet releases will contain both a release and pre-release version. The release version contains the public APIs we currently intend to release to general availability.
+## WebView2 Benefits 
+![webviewreasons](./media/WebView2/webviewreasons.PNG)
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:35]: There are differences between the icon titles and the text below. For example, Fixed vs. Fixed distribution. Can you review please?

File: microsoft-edge/webview2/index.md (hunk 1 above)
Comment (Reezaali) [reply microsoft-edge/webview2/index.md:37]: To follow the MS Style Guidelines, we should not use bold for emphasis.

File: microsoft-edge/webview2/index.md [hunk 11]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
-
-:::image type="complex" source="./media/vs-js-diagnostics.jpg" alt-text="Visual Studio JavaScript diagnostics":::
-   Visual Studio JavaScript diagnostics
-:::image-end:::  
-
-<!--![vs-js-diagnostics](./media/vs-js-diagnostics.jpg)  -->  
-
-Right click on your project and select **Properties**. Under **Configuration Properties** > **Debugging** > **Debugger Type**,  choose the **JavaScript (WebView2)** option to enable WebView2 script debugging. More details to follow soon.
+## Getting Started
+To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget)installed. Select one of the options below to get started!
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:48]:
```suggestion
To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget) installed. Select one of the options below to get started.
```

File: microsoft-edge/webview2/index.md [hunk 12]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
-:::image-end:::  
-
-<!--![vs-js-diagnostics](./media/vs-js-diagnostics.jpg)  -->  
-
-Right click on your project and select **Properties**. Under **Configuration Properties** > **Debugging** > **Debugger Type**,  choose the **JavaScript (WebView2)** option to enable WebView2 script debugging. More details to follow soon.
+## Getting Started
+To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget)installed. Select one of the options below to get started! 
+- [Getting Started with Win32 C/C++]()
+- [Getting Started with WPF]()
+- [Getting Started with WinForms]()
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:51]: The links are missing.

File: microsoft-edge/webview2/index.md [hunk 13]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-:::image type="complex" source="./media/vs-script-debugger.jpg" alt-text="Visual Studio script debugger":::
-   Visual Studio script debugger
-:::image-end:::  
+The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API use patterns. As we add more features to the WebView2 SDK, we will regularly update our sample applications.
+- [Win32 C/C++ Sample]()
+- [WPF Sample]()
+- [WinForms Sample]()
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:56]: The links are missing.

File: microsoft-edge/webview2/index.md [hunk 14]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+## Getting Started
+To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget)installed. Select one of the options below to get started! 
+- [Getting Started with Win32 C/C++]()
+- [Getting Started with WPF]()
+- [Getting Started with WinForms]()
 
-:::image type="complex" source="./media/vs-script-debugger.jpg" alt-text="Visual Studio script debugger":::
-   Visual Studio script debugger
-:::image-end:::  
+The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API use patterns. As we add more features to the WebView2 SDK, we will regularly update our sample applications.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:53]:
```suggestion
The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API usage patterns. As we add more features to the WebView2 SDK, we will update our sample applications.
```

File: microsoft-edge/webview2/index.md [hunk 15]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+- [Win32 C/C++ Sample]()
+- [WPF Sample]()
+- [WinForms Sample]()
 
-<!--![vs-script-debugger](./media/vs-script-debugger.jpg)  -->  
+## Next Steps
+For more thorough information on how to build and deploy WebView2 application checkout our conceptual documentation and how-to guides.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:59]:
```suggestion
For more detailed information on how to build and deploy WebView2 applications, checkout our conceptual documentation and how-to guides.
```

File: microsoft-edge/webview2/index.md [hunk 16]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-### Visual Studio Code
+### Concepts
+1. [WebView2 SDK and Microsoft Edge Versioning]()
+2. [Distributing WebView2 Applications]()
+3. [Utilizing and Disabling Microsoft Edge Browser Features in WebView2]()
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:64]: TODO: add links to published docs when they go live.

File: microsoft-edge/webview2/index.md [hunk 17]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-You can also use Visual Studio Code to debug your script within the WebView2 right from the IDE. For more details click [here](https://github.com/microsoft/vscode-edge-debug2/blob/master/README.md#microsoft-edge-chromium-webview-applications).
+### How-To Guides
+1. [Debugging WebView2 with DevTools and Visual Studio Script Debugging]()
+2. [Automating and Debugging WebView2 with Microsoft EdgeDriver]()
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:68]: TODO: add links to published docs when they go live.

File: microsoft-edge/webview2/index.md [hunk 18]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-## Getting in touch with the WebView2 team  
+## Supported Platforms
+A developer preview is available for Win32 C/C++, .NET Framework 4.6.2 or later, .NET Core 3.0 or later, [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:71]:
```suggestion
A developer preview is available for: 

* Win32 C/C++.
* .NET Framework 4.6.2 or later.
* .NET Core 3.0 or later.
* [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2. 
```

File: microsoft-edge/webview2/index.md [hunk 19]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
 
-## Getting in touch with the WebView2 team  
+## Supported Platforms
+A developer preview is available for Win32 C/C++, .NET Framework 4.6.2 or later, .NET Core 3.0 or later, [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2. 
 
-Help us build a richer WebView2 experience by sharing your feedback! Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports.
+## Feedback
 
-It's also a good place to search for known issues.
+Help us build a richer WebView2 experience by sharing your feedback! Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports. It’s also a good place to search for known issues.
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:75]:
```suggestion
Help us build a richer WebView2 experience by sharing your feedback. Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports. It’s also a good place to search for known issues.
```

File: microsoft-edge/webview2/index.md [hunk 20]
```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
...
+### How-To Guides
+1. [Debugging WebView2 with DevTools and Visual Studio Script Debugging]()
+2. [Automating and Debugging WebView2 with Microsoft EdgeDriver]()
 
-## Getting in touch with the WebView2 team  
+## Supported Platforms
+A developer preview is available for Win32 C/C++, .NET Framework 4.6.2 or later, .NET Core 3.0 or later, [WinUI 3.0](https://docs.microsoft.com/uwp/toolkits/winui3/) on Windows 10, Windows 8.1, Windows 8, Windows 7, Windows Server 2016, Windows Server 2012/2012R2, and Windows Server 2008 R2. 
 
-Help us build a richer WebView2 experience by sharing your feedback! Visit our [feedback repo](https://aka.ms/webviewfeedback) to submit feature requests or bug reports.
+## Feedback
```
Comment (Reezaali) [inline microsoft-edge/webview2/index.md:73]:
```suggestion
## Getting in touch with the WebView2 team
```

File: microsoft-edge/webview2/index.md (hunk 10 above)
Comment (pagoe-msft) [reply microsoft-edge/webview2/index.md:33]: Josh is going to finalize these two titles in his distribution doc - will fix when his change is complete

File: microsoft-edge/webview2/index.md [hunk 21]
```diff
@@ -1,108 +1,157 @@
...
-## App Distribution
+*   [Getting Started with Win32 C/C++](./gettingstarted/win32.md)  
+*   [Getting Started with WPF](./gettingstarted/wpf.md)  
+*   [Getting Started with WinForms](./gettingstarted/winforms.md)  
 
-The WebView2 control utilizes the Microsoft Edge (Chromium) browser. When distributing your app it is important to ensure that the Edge browser is installed on all user machines where your application will run. You should also be aware of which version and channel is installed, e.g. Stable, Beta, Dev, or Canary. The WebView2 controller will utilize the most stable version of the browser when installed on a machine.
+The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDKs features and any API usage patterns.  As the Microsoft Edge WebView team adds more features to the WebView2 SDK, the Microsoft Edge WebView team updates the sample applications.  
 
-### Future Planned Changes
+*   [Win32 C/C++ Sample]()
```
Comment (jm-trd-ms) [inline microsoft-edge/webview2/index.md:93]: Please add the missing links.

File: microsoft-edge/webview2/index.md (hunk 16 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/index.md]: Updated.

File: microsoft-edge/webview2/index.md (hunk 17 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/index.md]: Updated.

File: microsoft-edge/webview2/index.md (hunk 12 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/index.md]: Updated

File: microsoft-edge/webview2/index.md (hunk 13 above)
Comment (jm-trd-ms) [reply microsoft-edge/webview2/index.md]: Updated

**Comment by jm-trd-ms** [comment]
#sign-off and merged.

**Pull request merged**